### DISCUSSION TITLE
Removing the VPN config

### DISCUSSION THREAD

**Discussion by sbaileys** [frkn-dev/FRKN#135] [Q&A]
Hi all! 
I have a macbook, and normally remove vpn services etc from the profiles section. But for this vpn it is disabled. So I am wondering if anyone has/can provide instructions on hot to delete the configuration for this vpn.

**Comment by EsDmitrii** [comment]
Privacy and security --> Profiles --> Delete profile on the screenshot
<img width="473" alt="Screenshot 2024-01-12 at 12 32 11" src="https://github.com/frkn-dev/FRKN/assets/124890665/985fd384-b360-4596-9c68-e06132180c27">
